source/frontend/src/features/mapSideBar/consolidation/AddConsolidationContainer.tsx

```diff
@@ -76,7 +76,9 @@ const AddConsolidationContainer: React.FC<IAddConsolidationContainerProps> = ({
           featuresetToMapProperty(selectedFeatureDataset),
         );
         if (isValidString(propertyForm.pid)) {
-          propertyForm.address = await getAddress(propertyForm.pid);
```

Comment:
This was a wasted of time and inaccurate, as we have this property in our system we should not be looking up the bca address for it.